vp8_remb_simulcast.c

```diff
@@ -0,0 +1,571 @@
+/* New methods for an REMB based simulcast layer switching on a subscriber peerConnection
```

Comment:
Please use the same templating we have in other files for the initial comments part. That's used by Doxygen to automatically create pages out of source files, and the related detailed information that describes its contents.